noir-projects/aztec-nr/aztec/src/macros/aztec.nr

```diff
@@ -249,36 +264,45 @@ comptime fn generate_contract_library_method_compute_note_hash_and_nullifier() -
 }
 
 comptime fn generate_sync_private_state() -> Quoted {
-    // We obtain the `external` function on the next line instead of directly doing
-    // `#[aztec::macros::functions::external("utility")]` in the returned quote because the latter would result in
-    // the function attribute having the full path in the ABI. This is undesirable because we use the information in
-    // the ABI only to determine whether a function is `external("private")`, `external("public")`, or `external("utility")`.
-    let external = crate::macros::functions::external;
```

Comment:
Interestingly doing this as well for the `abi_utility` attribute is no longer necessary as I just get `abi_utility` in the function attributes even though I have here the full path.

I guess Noir changed behavior.